pootle/apps/pootle_store/models.py

```diff
@@ -356,6 +359,11 @@ class Unit(models.Model, base.TranslationUnit):
                                     db_index=True, related_name='reviewed')
     reviewed_on = models.DateTimeField(db_index=True, null=True)
 
+    # this is calculated from virtualfolders if installed and linked
+    priority = models.FloatField(
+        db_index=True, default=1,
+        validators=[MinValueValidator(0.9)])
```

Comment:
ok - im setting the min to 0

everyone is happy - just that there is not actually a way to set it to 0